main.go

```diff
@@ -37,6 +37,7 @@ var (
 	flagInsecureAllowHTTP  bool
 	flagCompression        string
 	flagVersion            bool
+	flagOCISupport         bool
```

Comment:
I think this is not consumed anymore in the PR. Either you forgot to scratch this out, or there are some changes missing from this PR.